File: Backend/Domain/Entities/Offer.cs
using Domain.Entities.UserEntities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public  class Offer
    {
        public string Description { get; set; }
        public DateTime Duration { get; set; }
        public int Price { get; set; }
        public bool IsApproved { get; set; } = false;
        public bool IsCompleted { get; set; }=false;
        public Worker Worker { get; set; }
        public string WorkerId {  get; set; }
        public Job Job { get; set; }
        public int JobId { get; set; }
    }
}

File: Client/src/app/shared/models/offer.ts
export interface Offer {
  description: string;
  duration: string;
  price: number;
  isApproved: boolean;
  isCompleted: boolean;
  jobId: number;
  rating: number;
  completedTasks: number;
  pictureUrl: string;
  fristName: string;
  lastName: string;
  workerEamil: string;
  jobTitle?: string;
  workerId?: string;
}

export type offerDto = {
  jobId: number;
  description: string;
  Price: number;
  Duration: string;
};
